libs/go/athenzutils/hostname.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2021 Verizon Media, Inc.
```

Comment:
Must be: Copyright Athenz Authors

Reply:
without a year? Or `// Copyright 202 1 Athenz Authors`?